src/brevitas/quant/solver/common.py

```diff
@@ -150,13 +150,6 @@ def learned_round_init(tracked_parameter_list):
         return torch.full(tracked_parameter_list[0].shape, 0.)
 
 
-class SolveTensorQuantFloatToIntImplFromEnum(ExtendedInjector):
```

Comment:
What happened here? Why is it being removed?

Reply:
If you check, it is defined twice. I guess something happened during a rebase/merge